.gitignore

```diff
@@ -2,6 +2,7 @@
 
 # IDE
 .idea
+.vscode
```

Comment:
nitpick: Git has an option to use a ["global gitignore"](https://sebastiandedeyne.com/setting-up-a-global-gitignore-file). That is where I add things specific to my development environment. I wouldn't add it here.

Reply:
Didn't know that NICE